helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -292,7 +292,8 @@ private val PATH_EXCLUDE_REASON_FOR_FILENAME = listOf(
     "setup.py" to BUILD_TOOL_OF,
     "test.js" to TEST_OF,
     "test.py" to TEST_OF,
-    "test_*.c" to BUILD_TOOL_OF
+    "test_*.c" to BUILD_TOOL_OF,
+    "versioneer.py" to BUILD_TOOL_OF
```

Comment:
Please add a reference to https://github.com/python-versioneer/python-versioneer (if that's the correct tool) to the commit message body so it becomes more apparent where this tools comes from and what it does.